ansible/server-state-playbooks/ome-dundeeomero/playbook.yml

```diff
@@ -164,8 +184,28 @@
       command: cp "{{ check_mk_postgres_plugin }}"  /usr/share/check-mk-agent/plugins/
       when: check_mk_postgres_plugin_st.stat.exists
 
+    - name: PostgreSQL Nightly Backups | Create the backups directory
+      become: yes
+      file:
+        path: "{{ omero_server_db_dumpdir_parent }}/{{ omero_server_db_dumpdir_name }}"
```

Comment:
Minor point, the `omero_server_` prefix is used by the omero-server role, so there's potential for confusion between vars specific to this playbook and those in the role.